### DISCUSSION TITLE
tab active change color text - FF 125.0.0.1 - lepton style 8.6.0

### DISCUSSION THREAD

**Discussion by alvaro991** [black7375/Firefox-UI-Fix#884] [Q&A] [answered]
hello
when I have many tabs open: the selected one is blue with a black url. The other tabs are blue with a white URL. You can see it in the photo attach.
This way I can't see the tab in the foreground, I have black text that doesn't stand out as all the tabs are blue. How do I change the text color of the foreground tab? or to change the blue background of the tab in the foreground? thanks
![color](https://github.com/black7375/Firefox-UI-Fix/assets/46437189/92a092a4-0663-4a46-9fb1-e16315628949)

**Comment by black7375** [comment]
As far as know, This phenomenon is caused by #880
It should be better if you install the latest commit, not the released version.

**Comment by alvaro991** [comment]
hi thanks for the reply

**what are i? "latest commit"**
I'm using the file:  Lepton-Proton-Style.zip   02/19/2024  8.6.0
I can't figure out what other files to download and where
thanks

**Reply by black7375** [reply, marked as answer]
You can use `From Latest Code` with follow document:
- https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide#2-download-files--unzip

**Comment by alvaro991** [comment]
HI
thanks for the link
yes I found how to download the files and I changed them.
your secure link is faster
I put the new files in and it seems to work fine. thanks

**Reply by black7375** [reply]
Once the serious-looking error in FFv126 #881 is fixed, I will release it quickly.
Thanks.
